.buildkite/gpu_pipeline/pipeline.yml

```
@@ -122,3 +123,11 @@ steps:
           slurm_gpus_per_task: 1
           slurm_cpus_per_task: 4
           slurm_ntasks: 4
+
+      - label: ":fire: Target EDMF Aquaplanet 1 thread"
+        key: "flame_longrun_aquaplanet_rhoe_equil_55km_nz63_clearsky_diagedmf_0M"
+        command:
+          - julia --color=yes --project=examples perf/longer_flame.jl config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_allsky_diagedmf_0M.yml
```

```suggestion
          - julia --color=yes --project=perf perf/longer_flame.jl config/longrun_configs/longrun_aquaplanet_rhoe_equil_55km_nz63_allsky_diagedmf_0M.yml
```

ProfileCanvas is in the perf env, if you want to just use that instead